node/runtime/src/lib.rs

```diff
@@ -710,7 +710,7 @@ impl Runtime {
         debug!(target: "runtime", "{}", log_str);
     }
 
-    fn process_transaction(
+    pub fn process_transaction(
```

Comment:
Why making this public?

Reply:
To reuse runtime when need to process one tx at a time.